src/utils/retry-error.rb

```diff
@@ -25,65 +24,44 @@
 #  to the completed folder, if fails it is not moved.
 
 # Make Sure that the ENV variables are set
-if (!ENV.include? 'HYDRA_PROJECT_NAME') then
+if !ENV.include? 'HYDRA_PROJECT_NAME'
```

Comment:
I hate rubo cop's styling with if/else blocks. IMO multi-line if should have parens and a then. But, if you are using rubo cop for linting, whatever. But. yuck. (imo).

Also, since you are following Rubocop style guide here, this should be a single line if (rubo cop prefers single line if whenever possible, which i do agree with :) )

`abort "Missing ENV variable, 'HYDRA_PROJECT_NAME'"` if !ENV.include? 'HYDRA_PROJECT_NAME'